help/C/Help_ch_Transactions.xml

```diff
@@ -1067,28 +1067,112 @@ Author:
-        will have a light green background and the A and U+R checkboxes will be unchecked and the R checkbox will be 
-        checked. To override and import the transaction, check the A checkbox. The U and R boxes will be unchecked 
-        automatically. The reliability of the match is indicated by a bar display in the Info column. If a destination
-        account for the second split is assigned by the matcher if will be appended to the info column.
-      </para>
-      <para>Transaction rows which do not match existing transactions in the import account, for which an assignment
-        of a destination account cannot be made on the basis of the previous import history to the account, will be 
-        displayed with an orange-yellow background and the A box will be checked and U+R and R unchecked. A destination 
-        account must be specified for these transactions.
+      <para>The status of the imported transaction rows is indicated by a combination of the background colour for the row and three
+       checkboxes in the columns labelled A (Add), U+R (Update and Reconcile) and R (Reconciled) as described in <xref linkend="match_status"/>.
       </para>
+      <note><para>Note that reconciled in this context is not the same as reconciled in the process of reconciling an account.</para></note>
+      <table id="match_status" frame='all'><title>Status of transactions in the Import Matcher</title>
+        <tgroup cols='3' align='left' colsep='1' rowsep='1'>
+        <colspec colname='c1'/>
+        <colspec colname='c2'/>
+        <colspec colname='c3'/>
+        <thead>
+          <row><entrytbl cols='2' colsep='1'><tbody><row><entry>Checkbox</entry><entry>State</entry></row></tbody></entrytbl><entry>Row Background Color</entry><entry>Meaning</entry></row>
+        </thead>
+        <tbody>
+          <row>
+            <entrytbl cols='2' align='left' colsep='1' rowsep='1'>
+                <tbody>                  
+                  <row><entry><emphasis role="strong">A</emphasis></entry><entry><emphasis>X</emphasis></entry></row>
+                  <row><entry><emphasis role="strong">U+R   </emphasis></entry><entry>-</entry></row>
+                  <row><entry><emphasis role="strong">R     </emphasis></entry><entry>-</entry></row>                  
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry><?dbhtml bgcolor="#FFCC00"?><?dbfo bgcolor="#FFCC00"?>Yellow-Gold</entry></row> <!-- FIXME need to check the color values against those used in GnuCh -->
+                <row><entry><?dbhtml bgcolor="#CCFFCC"?><?dbfo bgcolor="#CCFFCC"?>Light Green</entry></row>
+                </tbody>
+            </entrytbl>
+           <entrytbl cols='1'>
+                <tbody>
+                <row><entry>Flagged for input-transfer account must be selected.</entry></row>
+                <row><entry>Flagged for input-transfer account matched.</entry></row>
```

Comment:
How about typographical correct `Flagged for input&mdash;transfer account...` here and later
or more modern `Flagged for input &ndash; transfer account...`
or `Flagged for input - transfer account...` like below?

Reply:
Frank,

I will create the further mods in a new branch as you suggested earlier and submit a new PR when they are done.  Bob has provided a reference to the default gtk CSS which will help with getting the right default colors. It may take me a while to get the table structured so it comes up properly in all the various formats.